addons/test_mail/static/tests/properties_field_tests.js

```diff
@@ -0,0 +1,76 @@
+/** @odoo-module **/
+
+import { start, startServer, click } from "@mail/../tests/helpers/test_utils";
+
+QUnit.module("properties field");
+
+/**
+ * Open the chatter when clicking on an avatar many2one / many2many properties.
+ */
+async function testPropertiesOpenChatter(assert, propertyType) {
+    const pyEnv = await startServer();
+    const view = `
+        <form string="Simple">
+            <sheet>
+                <field name="name"/>
+                <field name="parent_id"/>
+                <field name="properties"/>
+            </sheet>
+            <div class="oe_chatter">
+                <field name="message_ids"/>
+            </div>
+        </form>`;
+
+    const { openView } = await start({
+        serverData: {
+            views: {
+                "mail.test.properties,false,form": view,
+            },
+        },
+        async mockRPC(route) {
+            if (route.includes("/mail.test.properties/check_access_rights")) {
+                return true;
+            } else if (route === "/web/dataset/call_kw/res.users/read") {
+                partnerFetched = true;
+                return [{ id: userId, partner_id: [partnerId, "Partner Test"] }];
+            } else if (route === "/web/dataset/call_kw/res.users/search_read") {
+                return [{ id: userId, name: "User Test" }];
+            }
+        },
+    });
+
+    let partnerFetched = false;
+
+    const partnerId = pyEnv["res.partner"].create({ name: "Partner Test" });
+    const userId = pyEnv["res.users"].create({ name: "User Test" });
+    const parentId = pyEnv["mail.test.properties"].create({ name: "Parent" });
+    const value = propertyType === "many2one" ? [userId, "User Test"] : [[userId, "User Test"]];
+    const recordId = pyEnv["mail.test.properties"].create({
+        name: "Test",
+        parent_id: parentId,
+        properties: [{type: propertyType, comodel: "res.users", name: "user", string: "user", value }],
+    });
+
+    await openView({
+        res_id: recordId,
+        res_model: "mail.test.properties",
+        views: [[false, "form"]],
+    });
+
+    assert.notOk(partnerFetched);
```

Comment:
Could we use `assert.step` and `assert.verifyStep` instead of using a boolean variable `partnerFetched`?

In the function mocking the rpc:

```js
async mockRPC(route) {
    if (route.includes("/mail.test.properties/check_access_rights")) {
        return true;
    } else if (route === "/web/dataset/call_kw/res.users/read") {
        assert.step("read res.users");
        return [{ id: userId, partner_id: [partnerId, "Partner Test"] }];
    //...
```

In the checks:
 
```js
assert.verifySteps([]);
await click(propertyType === "many2one" ? ".o_property_many2one_avatar" : ".o_m2m_avatar");
assert.verifySteps(["read res.users"]);
```